app/Console/Commands/Wiki/OrgDigitalBibleLibraryCompare.php

```diff
@@ -71,8 +71,8 @@ private function organizationExists($dbl_org)
             OrganizationRelationship::firstOrCreate([
                 'type'                   => 'Member',
                 'organization_child_id'  => $translationMatchExists->organization->id,
-                'organization_parent_id' => $this->dbl_id,
-                'relationship_id'        => $dbl_org->id
+                'organization_parent_id' => $this->dbl_id
+                // 'relationship_id'        => $dbl_org->id
```

Comment:
why commented out?

Reply:
I think that the comment line should be removed. The above because the `relationship_id` column doesn't exist in the current organization_relationships entity. So, it was throwing error when you try to use the command: `OrgDigitalBibleLibraryCompare`